openedx/frontend-app-authoring
# fix: find proper courses when searching with active/archived filters on

## Description

This PR makes the Studio Home search bar work as expected when are applied archived/active filters. When users attempt to filter for active/archived in combination with the search input, the proper courses are displayed.

## Supporting information

GitHub Issue: https://github.com/openedx/frontend-app-authoring/issues/1477

## Testing instructions

Make sure you count on at least one course and do the following:

1. Mount this branch on the `frontend-app-authoring` MFE to test it
2. Go to the [Studio Home](http://apps.local.openedx.io:2001/authoring/home), you'll watch all your courses
3. Now apply the filter `Archived` in the dropdown beside the search bar 
4. Try to look for a course you know is active, but if everything goes well, you shouldn't be able to find it
5. The same must happen if you have activated the filter `Active` and try to look for an archived course
6. Make sure you are shown the message "We could not find any results."

![image](https://github.com/user-attachments/assets/de8a7842-4e8a-458b-8ad0-6575e7ad9d6d)
